pkgs/build-support/trivial-builders.nix

```
@@ -541,4 +541,31 @@ rec {
       phases = "unpackPhase patchPhase installPhase";
       installPhase = "cp -R ./ $out";
     };
+
+  /* Checks the command output contains the specified version
+   *
```

```suggestion
  /* Checks the command output contains the specified version.
   *
   * Although simplistic, this test assures that the main program
   * can run. While there's no substitute for a real test case,
   * it does catch dynamic linking errors and such.
   *
```

Clarify the purpose, or at least why I would use it.